web_widget_one2many_product_picker/models/ir_http.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Tecnativa - Alexandre D. Díaz
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models, tools
+from odoo.http import request
+
+
+class Http(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def get_currencies(self):
```

Comment:
This seems a very low level place to put this. Can you do it other way? You already have the currency in the form, and you can pass it as option to the widget if needed (but I don't think so, as the default is to `currency_id` if I'm not wrong.

Reply:
Ok, now currencies are get from the widget.
Yes, by default uses "currency_id" and can be changed via options using "currency_field".